app/kubemci/cmd/create.go

```diff
@@ -129,6 +129,10 @@ func validateCreateArgs(options *CreateOptions, args []string) error {
 	if options.GCPProject == "" {
 		return fmt.Errorf("unexpected missing argument gcp-project.")
 	}
+
+	if options.KubeconfigFilename == "" {
+		fmt.Println("NOTE: Using the default kubeconfig file will create an ingress in all contexts. Depending on your setup, this may expose services you did not intend to expose.")
```

Comment:
s/NOTE/WARNING/

Reply:
After our discussion offline, this is no longer relevant, since the flag is now required.